Cargo.toml

```diff
@@ -1,2 +1,2 @@
 [workspace]
-members = [ "elements-harness", "swap", "elements-fun" ]
+members = [ "elements-harness", "swap", "elements-fun", "waves/src/native" ]
```

Comment:
Is `waves` really such a good name considering there is a whole blockchain ecosystem named like that?

Reply:
Happy to rename it :) Back to Greek god names?